package.json

```diff
@@ -42,9 +42,9 @@
     "prettier": "prettier --write .",
     "prettier-watch": "onchange \"**/*.{js,ts,tsx}\" -- prettier --write --ignore-unknown {{changed}}",
     "print-version": "echo $npm_package_version",
-    "storybook": "start-storybook -p 6006",
-    "storybook-build": "ENV=production build-storybook -o dist/docs",
-    "storybook-build-staging": "ENV=staging build-storybook -o dist/docs",
+    "storybook": "storybook dev -p 6006",
+    "storybook-build": "ENV=production storybook build -o dist/docs",
+    "storybook-build-staging": "ENV=staging storybook build -o dist/docs",
```

Comment:
Did you try building it to see if this is what caused the storybook not to load?